tests/queries/0_stateless/03315_executable_table_function_threads.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest, no-debug, no-asan, no-tsan, no-msan, no-ubsan
+
+set -e
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
```

Comment:
(someone looking at this file in future will have a hard time figuring out what is actually tested. in future, please test with a [minimal reproducible example](https://en.wikipedia.org/wiki/Minimal_reproducible_example) and add some comments to give the reader context, thanks)

Reply:
Okay, thanks for clarifying and sorry then.